test/runtests.jl

```diff
@@ -52,6 +52,8 @@ end
                 0 3 6 9 0;
                 0 0 0 0 0]
     @test A == @inferred(PaddedView(0.0, a, (5, 5), (2, 2)))
+
+    @test PaddedView(0, ones(4), (3,), (1,)) == PaddedView(0, ones(4), (3,))
```

Comment:
I'm on board with this change and will merge a revised version. If we're deleting a sanity check, then we need more rigorous tests. Please:

- test with "parent" arrays that have different values in each slot, so you can be sure you're seeing the right values
- also test with OffsetArrays to make sure that it works properly with general indexing

Reply:
I think it's done.